go.mod

```diff
@@ -240,3 +240,5 @@ require (
 	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/minio/madmin-go/v3 v3.0.5 => github.com/shtripat/madmin-go/v3 v3.0.0-20230627103626-8485906f4a89
```

Comment:
Do you still need this?

Reply:
After merge of madmin-go PR merge, new version can be referred